src/lcm/error_handling.py

```diff
@@ -42,12 +44,30 @@ def validate_value_function_array_integrity(
         states=state_action_space.states,
         period=period,
     )
-
+    extrapolation = False
```

Comment:
Please call this `invalid_extrapolation` or so. In many cases, extrapolation beyond the grid is perfectly fine. E.g., our last gridpoint on the consumption grid could be 1M; if then we end up with the founder of a Unicorn company in our simulations, she will likely consume more, which would be perfectly fine ofc...